metabase/metabase
# Add approved domains FE validation

Fixes https://github.com/metabase/metabase/issues/18235

How to test:
- Run Metabase EE
- Go to Admin > General and set approved email domain
- Go to a dashboard, open the subscription sidebar and enter an email recipient with another domain
- Click Send now button
- You should see an error message that this domain is not allowed

Dashboard subscriptions:
<img width="396" alt="Screen Shot 2021-10-08 at 5 48 59 pm" src="https://user-images.githubusercontent.com/8542534/136582604-877761b9-d65d-4c2a-bfb7-107f294d9548.png">

Alerts:
<img width="599" alt="Screen Shot 2021-10-08 at 6 21 01 pm" src="https://user-images.githubusercontent.com/8542534/136582646-93a7343e-7881-45ba-b844-0326f43a4414.png">

Audit app:
<img width="668" alt="Screen Shot 2021-10-11 at 4 25 02 pm" src="https://user-images.githubusercontent.com/8542534/136797860-50ceda9d-3cfc-4935-8f71-07c91d0bc82e.png">
